python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -1233,3 +1263,47 @@ def _make_outputs(self, wcsf, visitSummaryTables, exposureInfo):
             catalogs[visit] = catalog
 
         return catalogs
+
+    def _output_model_params(self, wcsf):
+        """Get the WCS model parameters and covariance and convert to a
+        dictionary that will be readable as a pandas dataframe.
+
+        Parameters
+        ----------
+        wcsf : `wcsfit.WCSFit`
+            WCSFit object, assumed to have fit model.
+
+        Returns
+        -------
+        modelParams : `dict`
+            Parameters and covariance of the best-fit WCS model.
+        """
+        modelParamDict = wcsf.mapCollection.getParamDict()
+        modelCovariance = wcsf.getModelCovariance()
+
+        modelParams = {"mapName": [], "coordinate": [], "parameter": [], "coefficientNumber": []}
+        i = 0
+        for mapName, params in modelParamDict.items():
+            nCoeffs = len(params)
+            # There are an equal number of x and y coordinate parameters
+            nCoordCoeffs = nCoeffs // 2
+            modelParams["mapName"].extend([mapName] * nCoeffs)
+            modelParams["coordinate"].extend(["x"] * nCoordCoeffs)
+            modelParams["coordinate"].extend(["y"] * nCoordCoeffs)
+            modelParams["parameter"].extend(params)
+            modelParams["coefficientNumber"].extend(np.arange(nCoordCoeffs))
+            modelParams["coefficientNumber"].extend(np.arange(nCoordCoeffs))
+
+            for p in range(nCoeffs):
+                if p < nCoordCoeffs:
+                    coord = "x"
+                else:
+                    coord = "y"
+                modelParams[f"{mapName}_{coord}_{p}_cov"] = modelCovariance[i]
+                i += 1
```

Comment:
Was this just for debugging? The value never seems to be read.

Reply:
The `i`? It's in the line above.